__test__/reporter/absolute-path-reporter.spec.js

```diff
@@ -66,7 +66,7 @@ describe('AbsolutePathReporter', () => {
                         testResults: [
                             {
                                 fullName: 'Skipped "this test is skipped"',
-                                duration: 10,
```

Comment:
Add an additional testresultaten to test the null case, since we still need a test to cover the other case

Reply:
@sh33dafi addressed the feedback and added a new test case.